DirectEmployers/MyJobs
# [PD-3569] contact and location clarification on creation

## What's this PR do?
PR adds a clarification message to the NEW Cbo page and Additional Contact page

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the the TJD application

## How should this be manually tested?
Pull down the repository. Go into the CBO application and go to create a NEW CBO. You will see the message on the right hand side with the facets. As well go into a CBO to edit that is already created. Go and add an additional contact and you will also see the message on the right hand facet side.

## Any background context you want to provide?
https://directemployersfoundation.atlassian.net/browse/PD-3569
